systemtest/src/test/java/io/strimzi/systemtest/Resources.java

```diff
@@ -49,6 +50,10 @@
 import java.util.List;
 import java.util.stream.IntStream;
 
+import static io.strimzi.api.kafka.model.KafkaResources.entityOperatorDeploymentName;
+import static io.strimzi.api.kafka.model.KafkaResources.kafkaStatefulSetName;
+import static io.strimzi.api.kafka.model.KafkaResources.zookeeperStatefulSetName;
```

Comment:
I would not use static import for these ones as you are not using for the bootstrap connections in the same file.